brightchain-lib/src/lib/blocks/encryption.ts

```
@@ -0,0 +1,64 @@
+        ? BlockType.EncryptedConstituentBlockListBlock
+        : BlockType.EncryptedOwnedDataBlock,
+      BlockDataType.EncryptedData,
+      block.blockSize,
+      finalBuffer,
+      checksumService.calculateChecksum(finalBuffer),
+      creator,
+      block.dateCreated,
+      block.data.length,
+    );
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)